src/pkg/oci/common.go

```diff
@@ -24,33 +21,59 @@ import (
 )
 
 const (
-	// ZarfLayerMediaTypeBlob is the media type for all Zarf layers due to the range of possible content
-	ZarfLayerMediaTypeBlob = "application/vnd.zarf.layer.v1.blob"
-	// ZarfConfigMediaType is the media type for the manifest config
-	ZarfConfigMediaType = "application/vnd.zarf.config.v1+json"
-	// SkeletonArch is the architecture used for skeleton packages
-	SkeletonArch = "skeleton"
 	// MultiOS is the OS used for multi-platform packages
 	MultiOS = "multi"
 )
 
+// Logger is an interface built to
+type Logger interface {
```

Comment:
I noticed that we actually only use debug in this package, do we still want a logger interface for the future or should I just accept a single function?

Reply:
Agreed, and with that we need to just how our packages will display the logs. Args ..any could be slog style or printf style. Currently I have it as printf